t/app/controller/waste.t

```diff
@@ -64,12 +64,12 @@ FixMyStreet::override_config {
     };
     subtest 'Report a missed bin' => sub {
         $mech->content_contains('service-101', 'Can report, last collection was 27th');
-        $mech->content_contains('service-537', 'Can report, last collection was 27th');
+        $mech->content_lacks('service-537', 'Cannot report, last collection was 27th but the service unit has a report');
```

Comment:
Only thing I wonder is if we should check that it says someone has already reported it? Mostly as I wanted to double check it did this because saying that for flats seems particularly important.